pixl_ehr/tests/docker-compose.yml

```diff
@@ -58,10 +46,3 @@ services:
         N_TABLE_ROWS: 2
     ports:
       - "35433:5432"
-
-    healthcheck:
-      test: ["CMD", "pg_isready", "-U", "postgres"]
-      start_period: 30s
-      interval: 10s
-      timeout: 30s
-      retries: 5
```

Comment:
Why remove only the healthcheck? If we keep the postgres database, would probably still need this no?